src/components/CustomCollection/CustomCollection.tsx

```diff
@@ -58,6 +70,21 @@ const CustomCollection = ({
 }: PropTypes) => {
   const linkInput = useRef<ComboBoxRef>(null)
 
+  const [, setActiveDragId] = useState(null)
```

Comment:
Question: How does this get used? Does something else check the state variable outside of this file?

Reply:
Oh I forgot to remove this. dnd-kit has a component named `<DragOverlay />` that basically just lets you define some jsx and additional styles while dragging an item. So I can make the bookmark look completely different if I wanted to while dragging it. The `activeId` was just acting as a flag to display the `<DragOverlay />` with its children when a drag event starts.